src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const bookId = action.payload;
+      state.bookItems.push(bookId);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.bookItems = state.bookItems.filter((book) => book.id !== bookId);
+    },
+  },
+});
+
+// console.log(bookSlice);
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉

Reply:
Ok, I will remove it